bigbluebutton-html5/imports/ui/components/dropdown/list/item/component.jsx

```diff
@@ -58,7 +58,7 @@ export default class DropdownListItem extends Component {
         {
           description ?
           (<span id={this.descID} key="describedby" hidden>{description}</span>)
-          : null
+          : (<span id={this.descID} key="describedby" hidden></span>)
```

Comment:
You could just remove the if if you will add an empty element.

`<span id={this.descID} key="describedby" hidden>{description}</span>`